### ISSUE TITLE
Error generating France

### ISSUE THREAD

**Issue by emhansen** [evansiroky/timezone-boundary-builder#14]
I am getting an error when processing data for France (all preceding regions consistently work as expected).  The error is repeatable when resuming the script.  If I delete other regions generated json files (in downloads directory) and resume the script the deleted regions are correctly generated and the script fails on France again.

```
getting data for France
downloading from overpass
waiting 4 seconds
Success, decreasing overpass request gap
/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/jsts/dist/jsts.min.js:17
if(null!==t._sym||null!==e._sym||null!==t._next||null!==e._next)throw new IllegalStateException("Edges are already initialized");return t.init(e),t},Li.create=function(t,e){var n=new Li(t),i=new Li(e);return n.init(i),n},ro(wi,Li),io(wi.prototype,{mark:function(){this._isMarked=!0},setMark:function(t){this._isMarked=t},isMarked:function(){return this._isMarked},interfaces_:function(){return[]},getClass:function(){return wi}}),wi.setMarkBoth=function(t,e){t.setMark(e),t.sym().setMark(e)},wi.isMarked=function(t){return t.isMarked()},wi.setMark=function(t,e){t.setMark(e)},wi.markBoth=function(t){t.mark(),t.sym().mark()},wi.mark=function(t){t.mark()},io(Ri.prototype,{insert:function(t,e,n){var i=this.create(t,e);null!==n?n.insert(i):this.vertexMap.put(t,i);var r=this.vertexMap.get(e);return null!==r?r.insert(i.sym()):this.vertexMap.put(e,i.sym()),i},create:function(t,e){var n=this.createEdge(t),i=this.createEdge(e);return Li.

TypeError: Cannot read property 'getGeometryType' of undefined
    at Fi.write (/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/jsts/dist/jsts.min.js:17:10307)
    at qi.write (/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/jsts/dist/jsts.min.js:17:13980)
    at geomToGeoJsonString (/Users/eric_hansen/dev/timezone-boundary-builder/index.js:110:39)
    at asynclib.auto.saveSingleMultiPolygon (/Users/eric_hansen/dev/timezone-boundary-builder/index.js:187:38)
    at runTask (/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:1619:13)
    at /Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:1559:13
    at processQueue (/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:1569:13)
    at taskComplete (/Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:1588:9)
    at /Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:1612:17
    at /Users/eric_hansen/dev/timezone-boundary-builder/node_modules/async/dist/async.js:906:16
```

**Comment by evansiroky** [comment]
oi.  Well, this looks like a jsts bug.  However, it is very likely that some kind of geometry in the OSM data for France is resolving to some kind of invalid geojson which then causes the jsts bug.  It might be possible to isolate the geometry issue of the France OSM relation and then fix it in OSM and get rid of this issue.  I'll try to make a bug report to jsts regardless.

**Comment by evansiroky** [comment]
After further investigation, it looks like someone edited the relation for France and removed all of the `ISO3166-1` tags from all of the polygons in the relation!  The only things getting returned were points for labels, so that caused the undefined error.  

If you open `osmBoundarySources.json` and change the tag for France to `ISO3166-1:alpha2` it should work.

**Comment by evansiroky** [comment]
Fixed in 60b8fd8945c514caee85205bab111b1e969f224c
